docs/examples/data-access.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accessing Data\n",
+    "\n",
+    "There are many different conventions used to define unstructured grids (UGRID, SCRIP, etc.). These conventions often have different variable names used to describe the same variables.\n",
```

Comment:
Slight correction: "There are many different grid types ..."

Remove: "These conventions often have different variable names used to describe the same variables"